db/schema.rb

```diff
@@ -84,6 +95,43 @@
     t.index ["ingredient_id"], name: "index_inventories_on_ingredient_id"
   end
 
+  create_table "meals", force: :cascade do |t|
+    t.string "name"
+    t.integer "price", default: 0
+    t.boolean "active"
+    t.boolean "non_veg", default: false
+    t.text "description"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "orders", force: :cascade do |t|
```

Comment:
Why are these changes present in this PR?

Reply:
Removed, pushed it by mistake